app/jobs/api/issues_label_issue_job.rb

```diff
@@ -1,11 +1,18 @@
 module Api
   class IssuesLabelIssueJob < IssueEventJob
-    include IsPublishable
     timestamp ->(params) { params[:issue]['updated_at'] }
     action "issue_labeled"
+    cache_key ->(message) {
+      label_name = message[:payload][:label] ? message[:payload][:label]['name'] : ""
+
+      "#{message[:meta][:action]}.#{message[:meta][:user]["login"]}.#{message[:meta][:identifier]}.#{message[:meta][:timestamp]}.#{ label_name }"
```

Comment:
`cache_key` is a class method that will override the default `_cache_key` class_attribute

It's syntactic DSL sugar to override default behavior of the cache key for labels. 

So, I think it's already doing what you suggested

it might be confusing because it's not using parenthesis

```
cache_key(->(message) { "here.derp" })
```